src/main/java/org/sagebionetworks/web/client/widget/profile/UserProfileEditorWidgetViewImpl.java

```diff
@@ -213,8 +215,29 @@ public void setUsername(String username) {
   }
 
   @Override
-  public void setEmail(String email) {
-    this.email.setText(email);
+  public void setEmails(List<String> emails, String notificationEmail) {
```

Comment:
Changed the email textbox to a div container, and we just add paragraph elements (with tooltips) to it.
If it's the notification email, list it first and in bold.